frame/nfts/src/mock.rs

```diff
@@ -84,6 +84,12 @@ impl pallet_balances::Config for Test {
 	type ReserveIdentifier = [u8; 8];
 }
 
+impl Incrementable for u32 {
+	fn increment(&self) -> Self {
+		self.saturating_add(1)
```

Comment:
We can actually implement this in `lib.rs`, and include all the integer types. Maybe write a macro for this to lessen the amount of code duplication?

Reply:
`saturating_inc` already exists in sp-arithmetic.